src/components/Transfer.tsx

```diff
@@ -50,46 +57,25 @@ const Transfer = ({ className }: Props) => {
     setTransferInput(event.target.value);
   };
 
-  const onReceiverChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const receiver = event.target.value;
-    setReceiverInput(receiver);
-    setConnectedReceiver({
-      receiver,
-      setReceiverMessage
-    });
-  };
-
   if (!areApiReady) {
     return null;
   }
 
-  // TO-DO: Remove <br /> by proper margins
-
   return (
-    <>
+    <Container className={classes.container}>
       <h2>Transfer</h2>
-      <Container className={className}>
-        <div className="receiver">
-          <TextField fullWidth onChange={onReceiverChange} value={receiverInput} label="Receiver" variant="outlined" />
-        </div>
-        <p>{receiverMessage && `${receiverMessage}`}</p>
-
-        <br />
-        <TextField onChange={onChange} value={transferInput} label="Sender" variant="outlined" />
-
-        <br />
-        <Button variant="contained" disabled={isButtonDisabled()} onClick={sendLaneMessage}>
-          Send Transfer
-        </Button>
-        <p>{receiverAddress && estimatedFee && `Estimated source Fee: ${estimatedFee}`}</p>
-      </Container>
-    </>
+
+      <Receiver />
+
+      <TextField onChange={onChange} value={transferInput} label="Amount" variant="outlined" />
+
+      <Button variant="contained" disabled={isButtonDisabled()} onClick={sendLaneMessage}>
+        Transfer from {sourceChain} to {targetChain}
```

Comment:
```suggestion
       Send Bridge Message 
```
Current label is super confusing, the transfer is NOT between networks.

Reply:
it was actually on the design ... i can change it